core/invoker/Dockerfile

```diff
@@ -4,12 +4,16 @@
 FROM scala
 
 ENV DOCKER_VERSION 1.12.0
+ENV DOCKER_DOWNLOAD_URL https://get.docker.com/builds/Linux/x86_64/docker-${DOCKER_VERSION}.tgz
 
 RUN apk add --update openssl
 
 # Uncomment to fetch latest version of docker instead: RUN wget -qO- https://get.docker.com | sh
 # Install docker client
-RUN curl -sSL -o docker-${DOCKER_VERSION}.tgz https://get.docker.com/builds/Linux/x86_64/docker-${DOCKER_VERSION}.tgz && \
+RUN curl -sSL -o docker-${DOCKER_VERSION}.tgz ${DOCKER_DOWNLOAD_URL} && \
+curl -sSL -o docker-${DOCKER_VERSION}.tgz.sha256 ${DOCKER_DOWNLOAD_URL}.sha256 && \
```

Comment:
Should we inline the SHA256 i.e. refer it by its value rather than url?

Reply:
Yes you are right it is safer, pushed the change.